test/parallel/test-async-wrap-pop-id-during-load.js

```diff
@@ -15,7 +15,7 @@ const { spawnSync } = require('child_process');
 
 const ret = spawnSync(
   process.execPath,
-  ['--stack_size=50', __filename, 'async']
+  ['--stack_size=75', __filename, 'async']
 );
 assert.strictEqual(ret.status, 0);
```

Comment:
Fwiw, I think this would have been more easily debuggable if this assertion had come last and the assertion on L22 would print out `stderr` in its failure message – maybe we can fix up the test for that as well?

Reply:
Oh, actually, I had a branch to fix this and then didn't open a PR, but maybe it can be tacked onto this. IIRC, the change was this:

```js
assert.strictEqual(ret.status, 0,
                   `EXIT CODE: ${ret.status}, STDERR: ${ret.stderr}`);
```